openebs-nativek8s/pipelines/stages/4-cleanup/cluster-cleanup/cluster-cleanup

```diff
@@ -1,6 +1,53 @@
 #!/bin/bash
 set -x
 
-time="$(TZ=IST date)"
-current_time=$time
-echo $current_time
\ No newline at end of file
+pod() {
+echo $CI_JOB_ID
+###clone e2e-nativek8s-repo
+echo "cloning e2e-nativek8s repo*************"
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p 1658 'git clone https://github.com/mayadata-io/e2e-nativek8s.git'
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p 1658 'cd e2e-nativek8s && git checkout release-branch && bash openebs-nativek8s/pipelines/stages/4-cleanup/cluster-cleanup/cluster-cleanup node '"'$CI_JOB_ID'"''
+}
+
+node() {
+job_id=$(echo $1)
+
+git clone https://github.com/mayadata-io/litmus.git
+cd litmus
+
+# Replace the VM IP In csv file
+sed -i -e 's/10.12.22.10/10.34.1.162/g' \
+-e 's/10.12.22.11/10.34.1.163/g' \
+-e 's/10.12.22.12/10.34.1.164/g' \
+-e 's/10.12.22.13/10.34.1.165/g' k8s/on-prem/openshift-installer/ip.csv
+
+sed -i '/10.34.1.165/a \
+10.34.1.166 \
+10.34.1.167' k8s/on-prem/openshift-installer/ip.csv
+
+#Replace the Vm names in csv file
+sed -i -e 's/auto1/d2iq-master.mayalabs.io/g' \
+-e 's/auto2/d2iq-node1.mayalabs.io/g' \
```

Comment:
@w3aman  change the vm names and ip

Reply:
@nsathyaseelan changed the vm names. PTAL